themes/default/content/blog/stack-readme/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "Stack READMEs in the Pulumi Service"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-05-16T14:35:01-07:00
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: Starting today you can add a README to your Pulumi Service Stacks to store key links, CLI commands and documentation.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - evan-boyle
+    - devon-grove
+    - myles-haynes
+    - casey-huang
+    - meagan-cojocar
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Starting today, users can create [Stack READMEs](https://www.pulumi.com/docs/intro/pulumi-service/projects-and-stacks/#stack-readme) in the [Pulumi Service](https://app.pulumi.com) that dynamically update based on [Stack Outputs](https://www.pulumi.com/learn/building-with-pulumi/stack-outputs). Stack READMEs interpolate output variables on the stack (`${outputs.instances[0].ARN}`) so that each stack can construct links to dashboards, shell commands, and other pieces of documentation. All of this content stays up to date as you stand up new stacks, rename resources, and refactor your infrastructure.
+
+<!--more-->
+
+There are a lot of operational activities that happen around [Pulumi Stacks](https://www.pulumi.com/docs/intro/concepts/stack), but critical information is spread across many tools. Our [User Experience team](https://www.pulumi.com/blog/get-to-know-pulumis-ux-team) recently conducted user interviews and we repeatedly heard about users moving across many tools and web apps to manage their deployments- from their cloud Console, to the Pulumi Service User Interface (UI), to the CLI, and so on. By collecting resource outputs in the Pulumi Service UI, we are aiming to ease this friction and keep relevant Stack information in one place that is kept up to date automatically.
```

Comment:
I am still not understanding how the UX studies and what they heard is relevant here. I almost feel like you should either back up and explain the problem or go into more depth what you mean by "many tools and web apps to manage their deployments". I think there is something buried in there that makes it hard to outright understand how this feature works

Reply:
i also did not understand this connection. @meagancojocar are you referring to the insight that people need to have a strong need in order to be willing to change to a different context? or another insight?

Reply:
Removed this detail to improve clarity :)